### ISSUE TITLE
Create new release for 0.12.0

### ISSUE THREAD

**Issue by sliptype** [LukaJCB/purescript-snabbdom#3]
Really nice work with this, I appreciate the hard work!

I want to add this package to [`package-sets`](https://github.com/purescript/package-sets). I was wondering if you could create a new release to incorporate the changes made for the latest version of the purescript compiler?

Let me know if you are no longer maintaining this, I would be happy to help!

**Comment by LukaJCB** [comment]
Unfortunately I won't be having the time to do this soon, if you want to help maintaining I can definitely add you as a contributor to this Repo :)

**Comment by sliptype** [comment]
Yes that would be great, I'll happily be a contributor and knock this out
